wpears/financial-well-being
# Add basic cf.gov header and footer

Add a basic cf.gov header and footer to the prototype.

To test:

1. Start a server with python -m SimpleHTTPServer
2. In a new terminal tab, run gulp clean && gulp build
3. Run gulp watch
4. A page should pop up in your browser with a cf.gov header and footer. The header mega menus won't work, but it should look like a cf.gov page.
